src/controllers/movies.js

```diff
@@ -0,0 +1,63 @@
+const { Prisma } = require('@prisma/client')
+const prisma = require('../utils/prisma')
+
+const getMovieList = async (req, res) => {
+	const movies = await prisma.movie.findMany({
+		orderBy: {
+			id: 'asc',
+		},
+		include: {
+			screenings: true,
+		},
+	})
+	return res.status(200).json({ movies: movies })
+}
+
+const createMovie = async (req, res) => {
+	const body = req.body
+	const newMovie = await prisma.movie.create({
+		data: {
+			title: body.title,
+			runtimeMins: body.runtimeMins,
+		},
+		include: {
+			screenings: true,
+		},
+	})
+	res.status(201).json({ movie: newMovie })
+}
+
+// by id
+const getById = async (req, res) => {
+	const id = Number(req.params.id)
+	const newMovie = await prisma.movie.findUnique({
+		where: {
+			id: id,
+		},
+		include: {
+			screenings: true,
+		},
+	})
+	res.status(200).json({ movie: newMovie })
+}
+
+// update movie
+const updateMovie = async (req, res) => {
+	const id = Number(req.params.id)
+	const body = req.body
+	const updatedM = await prisma.movie.update({
```

Comment:
updatedM could be a little ambiguous as to what it is referring to?

Reply:
UpdatedMovie, I didn't want to give the same name again and I just shortened it. I'm not very good at giving name to variables ahah 
I will consider that the next time 😊